keyboards/bastardkb/charybdis/3x5/rules.mk

```diff
@@ -0,0 +1,11 @@
+# Build Options
+#   change yes to no to disable
+#
+RGB_MATRIX_SUPPORTED = yes  # RGB matrix is supported and enabled by default
+RGBLIGHT_SUPPORTED = yes    # RGB underglow is supported, but not enabled by default
+
+SPLIT_KEYBOARD = yes
+
```

Comment:
This has previously been migrated and should not be reverted.
```suggestion
```

Reply:
Thank you for catching the accidental reversals. I guess I was a little eager once I finally had the conflicts resolved, sorry!